metrics/api/src/main/java/io/helidon/metrics/api/ScopeConfigBlueprint.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.metrics.api;
+
+import java.util.Optional;
+
+import io.helidon.builder.api.Prototype;
+import io.helidon.config.metadata.Configured;
+import io.helidon.config.metadata.ConfiguredOption;
+import io.helidon.inject.configdriven.api.ConfigBean;
+
+/**
+ * Configuration settings for a scope within the {@value MetricsConfigBlueprint#METRICS_CONFIG_KEY} config section.
+ */
+@ConfigBean()
+@Configured(prefix = "scopes")
```

Comment:
if this is supposed to be a root configuration, then the prefix should be `metrics.scopes` or similar. Otherwise the prefix should not be defined.

Reply:
Also this cannot be a `ConfigBean` if not `root`, so please remove that annotation as well.

Reply:
both done